docs/dokka.tree

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE product-profile
+        SYSTEM "https://resources.jetbrains.com/stardust/product-profile.dtd">
+
+<product-profile id="dokka"
+                 name="Dokka"
+                 start-page="quickstart.md">
+
+    <toc-element id="overview.md"/>
+    <toc-element toc-title="Runners">
```

Comment:
Maybe there's a better word than "Runners" that covers both build system plugins and the CLI alternative? Coulnd't come up with anything else